linode/helper/framework_resource_base.go

```diff
@@ -94,8 +143,6 @@ func (r *BaseResource) Schema(
 	resp.Schema = *r.Config.Schema
 }
 
-// ImportState should be overridden for resources with
-// complex read logic (e.g. parent ID).
```

Comment:
May worth still keeping this comment as a reminder? Just in case we still want to override the function for other complex logic.

Reply:
Fixed, thanks!